app/cdap/components/shared/ConfigurationGroup/WidgetWrapper/index.tsx

```diff
@@ -160,6 +161,7 @@ const WidgetWrapperView: React.FC<IWidgetWrapperProps> = ({
           disabled={disabled}
           errors={errors}
           dataCy={pluginProperty.name}
+          dataTestId={pluginProperty.name}
```

Comment:
Should we change components to only accept `dataTestId` now? Then have the component set both `data-cy` and `data-test-id` on the DOM element. Not a big difference but would save a little bit of cleanup later.

Reply:
Ideally yes we'd want that to reduce cleanup work, but in some cases I don't think it's doable. For example in `app/cdap/components/AbstractWidget/StateWrapper.js`, `dataCy` is passed to various components depending on `Comp` value and removing `dataCy` in one abstract widget component and not the other might start to get confusing/messy. I think removing all usages of `dataCy` in one PR might be a better way to handle it.